src/debian/snmp_smartctl.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
 /usr/sbin/smartctl --scan | grep -E "^/dev/(sd|nvme)" | awk '{ print $1 }' | while read i; do
-  echo ------------------------------------------
   echo $i
-  echo ------------------------------------------
-  /usr/sbin/smartctl -a $i
+  j=$(echo $i | sed "s/\/dev\///")
```

Comment:
Move this in another commit.

Reply:
oops, I forgot I had this in my working dir when pushing the above requested changes.
That's fixed.